### DISCUSSION TITLE
[Question]: What is Explosion Karma and Damage Karma?

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#1816] [Q&A] [answered]
### Which feature

This feature was added in #641 
What purpose it serves just wanted to know?

### Reason

It is present. want to know the use case.

### Additional context

![image](https://github.com/YimMenu/YimMenu/assets/139460769/bebdd6c0-9587-4b3e-853b-d023a03dc4af)

**Comment by lonelybud** [comment]
@maybegreat48 ?

**Reply by maybegreat48** [reply, marked as answer]
Explosion karma reflects explosions and damage karma reflects incoming damage back to the attacker (like the Minecraft thorns enchantment)

**Comment by lonelybud** [comment]
Thanks @xiaoxiao921 . i forgot discusions were available. Thank you
